extension/public/popup.css

```diff
@@ -258,6 +259,15 @@ header {
   background-color: rgba(255, 255, 255, 0);
 }
 
+/* Tag Item's Text */
+.keywords .tag-list .tag-text{
+    max-width: 124px;
+    text-overflow: ellipsis;
```

Comment:
nit pick: indentation. The indentation for these changes is different to the indentation in the rest of the files